packages/core/src/internal/services/i18n.service.ts

```diff
@@ -33,6 +33,13 @@ export interface I18nStrings {
     showButtonAriaLabel: string;
     hideButtonAriaLabel: string;
   };
+  pagination: {
+    paginationAriaLabel: string;
+    firstButtonAriaLabel: string;
+    prevButtonAriaLabel: string;
+    nextButtonAriaLabel: string;
+    lastButtonAriaLabel: string;
+  };
```

Comment:
Hi Shijir,
There is no need to add these to i18n service unless the aria labels are INSIDE the template of the component and thus inaccessible to customize by our application developers. In the pagination case, they can easily do this without us having these as properties in the component. For example, this can be done without it being an explicit property of the pagination / pagination button components:

```
 <cds-pagination aria-label="pagination">
      <cds-pagination-button aria-label="go to first page" action="first" disabled></cds-pagination-button>
      <cds-pagination-button aria-label="go to previous page" action="prev" disabled></cds-pagination-button>
      <span aria-label="current page">1 / 3</span>
      <cds-pagination-button aria-label="go to next page" action="next"></cds-pagination-button>
      <cds-pagination-button aria-label="go to last page" action="last"></cds-pagination-button>
</cds-pagination>
```

If a11y team recommends we have a default value, we should maybe have a broader discussion since we don't always follow this pattern of pre-setting a default value. In any case, this can be achieved without using the i18n service since these aria-labels are on the host element and not inside the template.

Reply:
This is an important question. Let's discuss this on the chat.